index.js

```diff
@@ -431,6 +445,30 @@ class GoogleAuth extends DbBase {
         }
       )
     })
+
+    if (dailyLimitConfig) {
+      createdUser.dailyLimitConfig = this._convertDailyLimitsArrayToObject(
+        await (
+          Promise.all(
+            Object.keys(dailyLimitConfig || []).map((category) => {
+              return new Promise((resolve, reject) => {
+                const { alert, block } = dailyLimitConfig[category]
+                this.db.run(
+                  `INSERT INTO ${DB_TABLES.ADMIN_USER_DAILY_LIMITS} (admin_id, category, alert, block) VALUES (?, ?, ?, ?)`,
```

Comment:
there is already similar function but with handling conflicts https://github.com/bitfinexcom/bfx-facs-auth-google/pull/47/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R561
maybe we can re-use it ?